mne_bids/tests/test_mne_bids.py

```diff
@@ -169,10 +202,22 @@ def test_edf():
 
     raw_to_bids(subject_id=subject_id, session_id=session_id, run=run,
                 task=task, raw_file=raw, output_path=output_path,
-                overwrite=True, kind='eeg')
+                overwrite='overwrite', kind='eeg')
+    raw_to_bids(subject_id=subject_id, session_id=session_id, run=run2,
+                task=task, raw_file=raw, output_path=output_path,
+                overwrite='overwrite', kind='eeg')
     cmd = ['bids-validator', '--bep006', output_path]
     run_subprocess(cmd, shell=shell)
 
+    # check that the scans list contains two scans
+    scans_tsv = make_bids_filename(
+        subject=subject_id, session=session_id, suffix='scans.tsv',
+        prefix=op.join(output_path, 'sub-01/ses-01'))
+    assert op.exists(scans_tsv)
+    if op.exists(scans_tsv):
```

Comment:
why do you need to do this? Shouldn't it exist?

Reply:
@monkeyman192 see above

Reply:
@monkeyman192 was this resolved?

Reply:
happy to merge once this is resolved